### DISCUSSION TITLE
Spawner objects inherited in new room

### DISCUSSION THREAD

**Discussion by qiangjiadong** [mozilla/hubs#5309] [General Questions & Discussion]
I found my interacted spawner objects were still existing in the same position when I go to a new room by Enable Fast Room Switching from preference. Any ideas on it?

Thks!

**Comment by matthewbcool** [comment]
You found a fast room switching bug. I don't think we have this one documented in the hubs client issues yet. Could you please help us document this bug here: https://github.com/mozilla/hubs/issues/new?assignees=&labels=bug%2C+needs+triage&template=bug_report.md&title=

Thank you for the report!
